src/main/java/uk/gov/digital/ho/hocs/queue/common/MigrationQueueListener.java

```diff
@@ -17,27 +19,37 @@ public class MigrationQueueListener {
 
     private final List<MessageHandler> queueMessageHandlers;
 
+    private final MessageLogService messageLogService;
+
     private final boolean shouldIgnoreMessages;
 
     public MigrationQueueListener(List<MessageHandler> queueMessageHandlers,
+                                  MessageLogService messageLogService,
                                   @Value("${aws.sqs.ignore-messages:false}") boolean shouldIgnoreMessages) {
         this.queueMessageHandlers = queueMessageHandlers;
+        this.messageLogService = messageLogService;
         this.shouldIgnoreMessages = shouldIgnoreMessages;
     }
 
     @SqsListener(value = "${aws.sqs.case-migrator.url}", deletionPolicy = SqsMessageDeletionPolicy.NO_REDRIVE)
```

Comment:
This could be set through `cloud.aws.sqs.handler.default-deletion-policy` in a propertys file, allowing for the removal of specific Listeners.

Reply:
OOS for now